ql/src/test/queries/clientpositive/create_like2.q

```diff
@@ -7,3 +7,34 @@ ALTER TABLE table1_n20 SET TBLPROPERTIES ('a'='1', 'b'='2', 'c'='3', 'd' = '4');
 SET hive.ddl.createtablelike.properties.whitelist=a,c,D;
 CREATE TABLE table2_n14 LIKE table1_n20;
 DESC FORMATTED table2_n14;
+
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.strict.managed.tables=true;
+set hive.create.as.acid=true;
+set hive.create.as.insert.only=true;
+
+create table test_mm(empno int, name string) partitioned by(dept string) stored as orc tblproperties('transactional'='true', 'transactional_properties'='default');
+desc formatted test_mm;
+
+-- Conversion from MM to External
+create external table test_external like test_mm LOCATION '${system:test.tmp.dir}/create_like_mm_to_external';
+desc formatted test_external;
+
+-- Conversion from External to MM
+create table test_mm1 like test_external;
+desc formatted test_mm1;
+
+-- Conversion from External to External
+create external table test_external1 like test_external;
+desc formatted test_external1;
+
+-- Conversion from mm to mm
+create table test_mm2 like test_mm;
+desc formatted test_mm2;
+
```

Comment:
The pull request title says that we have a problem with converting from mm to external tables. The tests convert from external tables to mm. Maybe fix the jira description?  Or add more tests?

Reply:
I will change the title, both MM to External & External to MM are failing.

Reply:
Last question: What happens when we run:
```
create external table test_external2 like test_mm;
```

Thanks,
Peter

Reply:
We are removing "EXTERNAL" from tblproperties, but not setting TableType as managed, so it fails with same exception. That is fixed in this patch @ CreateTableLikeOperation